opencog/nlp/learn/README

```diff
@@ -286,35 +286,31 @@ alpha-pages after all that work)
     cp -pr alpha-pages beta-pages
 ```
 * Review the file `opencog/nlp/learn/run/README`, and then copy the
-  scripts into your working dir. 
+  scripts into your working dir, if you didn't do it before.
 * Modify as needed for chosen language. Most of them don't need modification,
   by default, unless you want to do something unusual.
 
-  -- `run-all-servers.sh` runs the cogserver for
-     one language, in a multi-paneled `byobu` window.
-     It doesn't start the final batch script `./wiki-ss-<lang>.sh`,
-     which needs to be started by hand (for the <lang> that you want to
-     process). Use F4 to move to the 'parse' tab and start it there.
-
   -- `wiki-ss-en.sh` performs batch processing of input text files.
      This calls `ss-one.sh` to process one text file.
 
   -- `ss-one.sh` calls `split-sentences.pl` to split the text file into
      sentences, and then calls `submit-one.pl` to send the sentences
      for parsing. None of these should need any modification.
 
-  -- `pair-count-lang.scm` starts the cogserver and opens the 
-      database. You need to modify 
+  -- `observe-launch.scm` starts the cogserver and opens the 
+      database. You need modify the opencog-??.conf it reads 
+      (if not English), and pass the database details.
+
+* Run `observe-launch.scm` in your working directory.
       ```
-          (sql-open "postgres:///en_pairs?user=ubuntu&password=asdf")
+        guile -l observe-launch.scm  -- --db ??-pairs --user your_user --password asdf
       ```
-      to contain your database name, username and password (if needed)
+      (--user and --password are optional, as explained above).
 
-* Run `run-all-servers.sh` in your working directory.
-* In the byobu terminal, navigate to the 'parse' tab (with F4),
-  go back to the input text dir (the dir that contains `beta-pages`), and
+* In a separate terminal (same working directory),
```

Comment:
No not in a separate terminal. Run it all in byobu. If you have five different sessions going at once, and each secsion takes 3 or 4 or 5 windows -- you get 25 windows .. that would be a horrible disaster.  Its all got to fit into one window, else its just not scalable.